apps/src/templates/sectionProgress/LoadMoreProgressButton.story.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import LoadMoreProgressButton from './LoadMoreProgressButton';
+
+export default storybook => {
+  return storybook
+    .storiesOf('Progress/LoadMoreProgressButton', module)
+    .addStoryTable([
+      {
+        name: 'Load more student progress',
+        description: `Used on the teacher dashboard progress tab to load more student data`,
```

Comment:
This is more of a clarification question : I have observed that we use either back ticks or single quotes for description.  What is the preferred way to enclose a string?

Reply:
I'm actually not sure. I tend to use '' but I probably used `` because I copied this story from an existing one. Our styleguide doesn't seem to mention for JS, but you could ask in `#apps-dev`